gen/azure/cloud-config.yaml

```diff
@@ -74,7 +76,7 @@ bootcmd:
 disk_setup:
   ephemeral0:
     table_type: gpt
-    layout: [ 50, 50 ]
+    layout: [ 25, 50, 25 ]
```

Comment:
Ok, this might actually be a problem with some of the new instance types. We have another PR here: https://github.com/dcos/dcos/pull/844 to add support for new VM types (and also limit to only instances which have 2+ cores which is the min required for DC/OS).

According to https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-sizes/ the Standard_F2s only has an 8gb ephemeral disk... in which case we would need to allocate at least 10% for the tmp volume. I'm going to investigate a bit further to see if we can specify both absolute and percentage with the cloud-init disk_setup stanza.